datasets/nfts/nfts/datastore.py

```diff
@@ -433,3 +433,37 @@ def import_data(
     if source_offset is not None:
         delete_checkpoints(target_conn, event_type, commit=False)
         insert_checkpoint(target_conn, event_type, source_offset)
+
+
+def filter_data(sqlite_db: sqlite3.Connection, cli_args):
```

Comment:
Make this signature more clear. I would expect a signature like this:

```python
def filter_data(conn: sqlite3.Connection, start_time: Optional[int] = None, end_time: Optional[int] = None) -> None:
    ...
```

This makes it so much more clear what this function is doing. In general, don't use these arbitrary argument objects like `cli_args`. The whole reason we have a handler function in the cli module is to convert from `args` object to meaningful arguments.